docs/Manual/00 Introduction.md

```diff
@@ -2,40 +2,40 @@
+The above code expresses more than just the logic of matrix multiplication. It insists on a specific execution flow: first perform all the steps required to calculate `C(0,0)` in ascending order of `k`; then proceed to `C(0,1)`. However, in principle, a single order of execution should not be imposed because the iterations of this loop can be performed in any order while keeping the logic intact. Moreover, the above logic doesn’t utilize important optimization techniques, such as double-buffered caching or vectorization.
 
-In contrast, the Accera programming model draws a strict distinction between the logic and its implementation. Namely, the programmer first writes the logic using a pseudocode-like syntax, independent of the target platform and without any consideration for performance. After the abstract logic is specified, the programmer moves on to define the concrete implementation details.
+Accera on the other hand provides a strict distinction between the logic and its implementation. The programmer first implements the logic without performance considerations using a pseudocode-like syntax that is independent of the target platform. Once the logic is specified, only then does the programmer move to define the concrete implementation details. 
 
 ### 2: Mindfully trade-off safety versus expressivity
-The Accera programming model starts with a default implementation of the specified logic and allows the programmer to transform and manipulate that implementation in different ways. When used correctly, these transformations should be *safe*, which means that they do not influence the underlying logic. Using safe transformations allows the programmer to focus on performance, without having to worry about correctness. Moreover, safe transformations allow automatic search algorithms to search the space of transformations more aggressively, converge faster, and find better optima.
+Accera offers a programming model that provides a default implementation of the specified logic which can be transformed and manipulated in different ways. If used correctly, these transformations are *safe*, which means that the underlying logic remains intact. This allows the programmer to completely focus on the performance of the logic without worrying about its correctness. Moreover, these safe transformations allow automatic search algorithms to aggressively search for the space of transformations to converge faster and to find better optima. 
```

Comment:
“aggressively search the space” (because we are actually searching the parameter space, not searching for the space)